stories.json

```diff
@@ -31,7 +31,97 @@
+    "background": "northernlights"
+  },
+  {
+    "permalink": "agile",
+    "text": "Yes, we are an international team, yes, we are fully remote, yes, we are diverse when it comes to characters. But this was an evolving process. We went remote step by step and made our learnings, we started with Europe, as nobody has solved the time zone yet, and are now ready for the world. We did not hire a certain nationality in a row (like: Now, we open an office in the US); we did not only hire extroverts only. We took a lot of thoughts into ensuring that as different as people are, a few basic beliefs are shared. Now this scales.",
+    "background": "northernlights"
+  },
+  {
+    "permalink": "transparency",
+    "text": "Transparency starts at the interview. During interviews, a lot of things are said by the people who interview you. Most of the times only 50% are true. At Giant Swarm, I still haven't found anything.",
+    "background": "northernlights"
+  },
+  {
+    "permalink": "fail-fast",
+    "text": "'If it hurts, do more of it' - I told my dad that it's a fail-fast environment and he told me I should fit right in ;D",
+    "background": "northernlights"
+  },
+  {
+    "permalink": "unlimited-holidays",
+    "text": "Why we do not count holidays or work time? This goes back to the original founding of Giant Swarm, where we said that we need very good people and these people all do not work for money. Of course they need to be paid but they are all at a level where, on average, they do not move jobs for money, but rather for an interesting environment, good work topics, great people, and so forth. Self Actualisation if you will. They come to a company because they like the vision, the idea, and are intrinsically motivated to work there. As with most people, there are very few things I can do to closely monitor and control those people and it is already known that control is not a good motivator. Hence, my job as a “manager” needs to be to make you WANT to work. As a next step, if I manage to do that, and make sure to make it clear that everyone takes the rest they need, then I should just let you do that. Take rest, when and where you want. Especially in our current setting you might take your laptop to holidays and do a few hangouts from there, or read a business book on the weekend, play poker with friends and get one as a customer. The line between personal and work life blurs more and more and I want to support that, because it should and it should in a way that does not create stress. Quote: We all learned to answer our email on saturday morning but not to go to the cinema on tuesday at 14:00 - Ricardo Semler",
```

Comment:
```suggestion
    "text": "Why we do we not count holidays or work time? This goes back to the original founding of Giant Swarm, where we said that we need very good people and these people do not work only for money. Of-course they need to be paid but they are all at a level where, on average, they do not move jobs for money, but rather for an interesting environment, good work topics, great people, and so forth. Self-actualisation if you will. They come to a company because they like the vision, the idea, and are intrinsically motivated to work there. As with most people, there are very few things I can do to closely monitor and control those people and it is already known that control is not a good motivator. Hence, my job as a “manager” needs to be to make you WANT to work. As a next step, if I manage to do that, and make sure to make it clear that everyone takes the rest they need, then I should just let you do that. Take rest, when and where you want. Especially in our current setting you might take your laptop to holidays and do a few hangouts from there, or read a business book on the weekend, play poker with friends and get one as a customer. The line between personal and work life blurs more and more and I want to support that, because it should. But, it should in a way that does not create stress. Quote: We all learned to answer our email on saturday morning but not to go to the cinema on tuesday at 14:00 - Ricardo Semler",
```